File: trials/try.c
#include <stdio.h>
#include <stdlib.h>

struct node {
    int data;
    struct node* next;
};

int main() {
    // Create a linked list with three nodes
    struct node* head = (struct node*) malloc(sizeof(struct node));
    struct node* second = (struct node*) malloc(sizeof(struct node));
    struct node* third = (struct node*) malloc(sizeof(struct node));

    head->data = 1;
    head->next = second;

    second->data = 2;
    second->next = third;

    third->data = 3;
    third->next = NULL;

    // Iterate over the linked list
    struct node* current = head;
    while (current != NULL) {
        printf("%d ", current->data);
        current = current->next;
    }

    return 0;
}

File: trials/try.py
# import example

# print(example.helloworld())

class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

# Create a linked list with three nodes
head = Node(1)
second = Node(2)
third = Node(3)

head.next = second
second.next = third

# Iterate over the linked list
current = head
while current is not None:
    print(current.data, end=' ')
    current = current.next
